Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -6,46 +6,47 @@ Authors: Anne Baanen, Kexing Ying, Eric Wieser
 import Mathlib.LinearAlgebra.Matrix.Determinant.Basic
 import Mathlib.LinearAlgebra.Matrix.SesquilinearForm
 import Mathlib.LinearAlgebra.Matrix.Symmetric
+import Mathlib.LinearAlgebra.BilinearMap
 
 #align_import linear_algebra.quadratic_form.basic from "leanprover-community/mathlib"@"d11f435d4e34a6cea0a1797d6b625b0c170be845"
 
 /-!
-# Quadratic forms
+# Quadratic maps
 
-This file defines quadratic forms over a `R`-module `M`.
-A quadratic form on a commutative ring `R` is a map `Q : M → R` such that:
+This file defines quadratic maps over a `R`-module `M`.
+A quadratic map on a commutative ring `R` is a map `Q : M → N` such that:
 
-* `QuadraticForm.map_smul`: `Q (a • x) = a * a * Q x`
-* `QuadraticForm.polar_add_left`, `QuadraticForm.polar_add_right`,
-  `QuadraticForm.polar_smul_left`, `QuadraticForm.polar_smul_right`:
-  the map `QuadraticForm.polar Q := fun x y ↦ Q (x + y) - Q x - Q y` is bilinear.
+* `QuadraticMap.map_smul`: `Q (a • x) = a * a * Q x`
+* `QuadraticMap.polar_add_left`, `QuadraticMap.polar_add_right`,
+  `QuadraticMap.polar_smul_left`, `QuadraticMap.polar_smul_right`:
+  the map `QuadraticMap.polar Q := fun x y ↦ Q (x + y) - Q x - Q y` is bilinear.
 
 This notion generalizes to commutative semirings using the approach in [izhakian2016][] which
-requires that there be a (possibly non-unique) companion bilinear form `B` such that
-`∀ x y, Q (x + y) = Q x + Q y + B x y`. Over a ring, this `B` is precisely `QuadraticForm.polar Q`.
+requires that there be a (possibly non-unique) companion bilinear map `B` such that
+`∀ x y, Q (x + y) = Q x + Q y + B x y`. Over a ring, this `B` is precisely `QuadraticMap.polar Q`.
 
-To build a `QuadraticForm` from the `polar` axioms, use `QuadraticForm.ofPolar`.
+To build a `QuadraticMap` from the `polar` axioms, use `QuadraticMap.ofPolar`.
 
-Quadratic forms come with a scalar multiplication, `(a • Q) x = Q (a • x) = a * a * Q x`,
+Quadratic maps come with a scalar multiplication, `(a • Q) x = Q (a • x) = a * a * Q x`,
```

Comment:
I don't think the first equality is what is wanted here (this would not be a useful scalar multiplication, as it is not linear in `a`).

Reply:
```suggestion
Quadratic maps come with a scalar multiplication, `(a • Q) x = a • Q x`,
```